identity-hub-client/build.gradle.kts

```diff
@@ -13,8 +13,9 @@
  */
 
 plugins {
-    java
+    `java-library`
     id("org.openapi.generator") version "5.4.0"
+    `maven-publish`
```

Comment:
not needed at this point but should plan to include `signing` plugin for publishing to Maven Central.

Also will require tasks for including javadoc, source jars  and signing, can also be done at root level